lib/constants.js

```diff
@@ -481,6 +481,7 @@ export const DEFAULT_SETTINGS = {
   language: DEFAULT_LANGUAGE,
   fontFamily: 'Hack',
   fontSize: '14px',
+  lineHeight: '133%',
```

Comment:
The app currently uses `4 / 3` as the line height. This is the closest value that's an integer.